source/LumiCalReco/src/LumiCalClusterer_auxiliary.cpp

```diff
@@ -133,8 +134,10 @@ int LumiCalClustererClass::getNeighborId(int cellId, int neighborIndex) {
   GlobalMethodsClass::CellIdZPR( cellId, cellZ, cellPhi, cellR, arm);
 
   // change iRho cell index  according to the neighborIndex
-  if ( neighborIndex%2 ) cellR -= neighborIndex+1;
-  else cellR += neighborIndex+1;
+  if( neighborIndex >= 2 ){
+    if ( neighborIndex%2 ) cellR -= neighborIndex/2;
+    else cellR += neighborIndex/2;
+  }
 
```

Comment:
Yes, they are, here is code
root [0] int cellR = 0;
root [1] int i =0;
root [2] while ( i < 6 ) { 
cellR = ( i%2 ) ? cellR -= i/2 : cellR += i/2 ; 
cout << "cellR = " << cellR << endl; 
cellR = 0; i++; }
cellR = 0
cellR = 0
cellR = 1
cellR = -1
cellR = 2
cellR = -2
root [3] .q
If neighborIndex >3 , neighborhood extends along R only..